Core/Core/Extensions/UIFontExtensions.swift

```diff
@@ -45,9 +45,11 @@ public extension UIFont {
         case .regular16:
             return scaledFont(.body, for: .systemFont(ofSize: 16, weight: .regular))
         case .regular20:
-            return scaledFont(.callout, for: .systemFont(ofSize: 16, weight: .semibold))
+            return scaledFont(.callout, for: .systemFont(ofSize: 20, weight: .semibold))
```

Comment:
Looks like we are using `regular20` in a few places and now those labels are too big (see Quizzes list).

This should be called `semibold20` since the weight is `semibold`, not `regular`. And we already have a case for `semibold20`.

Can you do a search (cmd+shift+f) for `regular20` and replace them all with `semibold16`?

Or you can just revert this change and we can fix it later. Are you using `regular20` for pages list?